tests/test_core.py

```diff
@@ -363,7 +364,7 @@ def test_region_processing_skip_aggregation(model_name, region_names):
             [["m_a", "s_a", "World", "Primary Energy", "EJ/yr", 5, 6]],
             [
                 "Difference between original and aggregated data:",
-                "m_a   s_a      World  Primary Energy EJ/yr 2005         5           4",
+                "m_a   s_a      World  Primary Energy EJ/yr 2005         5",
```

Comment:
Why does this not check two values?

Reply:
It's because I added the relative difference column which makes the output too wide so now the warning message looks like this:
```console
2023-05-26 10:36:51 WARNING  Difference between original and aggregated data:
                                                 original  ...  difference (%)
model scenario region variable       unit  year            ...                
m_a   s_a      World  Primary Energy EJ/yr 2005         5  ...            20.0
```
I'm not super happy about it, I could change it to omit the _difference_ column and just put _original_ and _aggregated_.

Reply:
Indeed, but this depends anyway on the length of the variable name, right? Maybe `pd.set_option('display.max_columns', None)` is your friend?

Reply:
Thanks for the suggestion, it now wraps it around two lines:
```console
2023-05-30 14:02:56 WARNING  Difference between original and aggregated data:
                                                 original  aggregated  \
model scenario region variable       unit  year                         
m_a   s_a      World  Primary Energy EJ/yr 2005         5           4   

                                                 difference (%)  
model scenario region variable       unit  year                  
m_a   s_a      World  Primary Energy EJ/yr 2005            20.0  
```